src/Microsoft.DotNet.MSBuildSdkResolver/Interop.NETStandard.cs

```diff
@@ -16,6 +17,16 @@ internal static partial class Interop
     {
         internal static readonly bool RunningOnWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
 
+        internal static string realpath (string path)
+        {
+            var buffer = new StringBuilder(4096); // PATH_MAX from linux/limits.h
```

Comment:
@nguerrera yeah, I looked at that but I'm not 100% sure all the platforms Mono runs on support the POSIX.1-2008 behavior, that's why I took the "safe" approach (also free()-ing the string which was malloc'ed by realpath would introduce a bunch of additional boilerplate I guess).

Do you feel strongly about this?